screens/BooksScreen.js

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react';
+import { ActivityIndicator, FlatList, StyleSheet, Text, View } from 'react-native';
+
+export default function BooksScreen() {
+  const [isLoading, setLoading] = useState(true);
+  const [data, setData] = useState([]);
+
+  const getBooks = async () => {
+    try {
+      const response = await fetch('https://openlibrary.org/subjects/painting.json');
+      const json = await response.json();
+      setData(json.works);
+    } catch (error) {
+      console.error(error);
+    } finally {
+      setLoading(false);
+    }
+  }
+
+  useEffect(() => {
+    getBooks();
+  }, []);
+
+  //looking for ISBN
+  const availabilities =  data.map((book) => {
```

Comment:
your next step was to handle the isbn inside of the `renderItem` and no longer outside of it. why is it still here?

Reply:
thanks for pointing it out . I was stuck on this for way to long... now solved on 02bf2fed85b876743338afb291853980ce0fcc3f